installers/go-server/win/config/wrapper-server.conf

```diff
@@ -64,8 +68,8 @@ wrapper.java.additional.5=-Duser.language=en
 wrapper.java.additional.6=-Duser.country=US
 wrapper.java.additional.7="-Dcruise.config.dir=%CRUISE_SERVER_DIR%\config"
 wrapper.java.additional.8="-Dcruise.config.file=%CRUISE_SERVER_DIR%\config\cruise-config.xml"
-wrapper.java.additional.9=-DreservedForFuture.1
-wrapper.java.additional.10=-DreservedForFuture.2
+wrapper.java.additional.9=-Dcruise.server.port=%GO_SERVER_PORT%
+wrapper.java.additional.10=-Dcruise.server.ssl.port=%GO_SERVER_SSL_PORT%
```

Comment:
@akshaydewan — as discussed. can you inline the environment variables here and remove the declaration on line 50 so we can merge this PR?